examples/sampling_c3/urdf/frame/frame.sdf

```diff
@@ -0,0 +1,279 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<sdf version="1.7">
+  <model name="frame">
```

Comment:
Doesn't seem like this file was auto-generated with the latest version of the script -- it doesn't have the autogeneration comment.  Better to commit one that will match what your script will create.